modules/smart-agent_genericjmx/detectors-genericjmx.tf

```diff
@@ -35,6 +35,8 @@ EOF
     parameterized_subject = var.message_subject == "" ? local.rule_subject : var.message_subject
     parameterized_body    = var.message_body == "" ? local.rule_body : var.message_body
   }
+
+  max_delay = var.memory_heap_max_delay
```

Comment:
Not directly related to this PR, is it intentional to have no heartbeat detector for JMX metrics?